app/src/main/java/org/p2p/wallet/infrastructure/network/provider/SeedPhraseProvider.kt

```diff
@@ -30,7 +30,7 @@ class SeedPhraseProvider(
         set(value) {
             field = value
             saveSeedPhraseToStorage(value)
-            Timber.tag(TAG).i("updating user seed phrase: $value")
+            Timber.tag(TAG).i("updating user seed phrase: ${value.size}")
```

Comment:
remove `value.size`

Reply:
replaced with isEmpty to understand are we removing seedphrase from shared prefs

on reset we are doing seedPhrase = emptyList()